### PR TITLE
SIMSBIOHUB-426-2: Calculate and display submission regions

### PR DESCRIPTION
**Pull request by al-rosenthal**
## Links to Jira Tickets

[SIMSBIOHUB-426](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-426)

## Description of Changes

- Added migrations for `region_lookup` and `submission_regions`
- Added seed files for existing regions (taken from SIMS)
- Modified Submission fetch to include regions 
- Modified intake to include region calculations

## Testing Notes

- Run both SIMS and BioHub locally
- Create a Project/ Survey in SIMS 
- Publish that Survey
- In BioHub navigate to Submissions tab and see regions on new submission

### CODE REVIEW

**Review by KjartanE — APPROVED** [review]
Tested on local, works as expected!

File: database/src/migrations/release.0.8.0/fn_calculate_area_intersect.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+-- *******************************************************************
+-- Procedure: fn_calculate_area_intersect
+-- Purpose: This function accepts 2 geometries and calculates the area of intersection between them.
```
Comment (al-rosenthal) [inline database/src/migrations/release.0.8.0/fn_calculate_area_intersect.sql:3]: @MacQSL I've updated the comments here, let me know if this is still confusing

**Review by curtisupshall — APPROVED** [review]
Code looks good to me

**Review by MacDeluca — APPROVED** [review]
Looks gud!

**Pull request merged**